src/components/TheHeader.vue

```diff
@@ -1,10 +1,20 @@
 <template>
   <header>
-    <v-toolbar data-cy="nav-drawer" color="primary">
+    <v-toolbar data-cy="nav-drawer" :color="navToggle.isHomeView ? 'surface-2' : 'surface-1'">
       <v-toolbar-title>
         <div class="d-flex flex-md-row flex-row-reverse align-center justify-space-between" style="cursor: pointer">
-          <TheUserMenu />
+          <TheUserMenu :is-home-view="navToggle.isHomeView" />
```

Comment:
Same idea let's make this take a prop called `theme` that defaults to `'light'` and could alternatively be `'dark'` which is used to compute these colors?